proto/axelar/axelarnet/v1beta1/types.proto

```diff
@@ -51,3 +51,10 @@ message Asset {
     (gogoproto.nullable) = false
   ];
 }
+
+message Fee {
```

Comment:
Why isn't this not replacing https://github.com/axelarnetwork/axelar-core/blob/26ea9778585d5ed7fcad1a4bbb593d58661ce596/x/axelarnet/message_handler.go#L23?

Reply:
good to have a way to consolidate, the only reason prevent this is, given the call-contract fee amount uses sdk.coin while the message handler fee amount is a string due to the ICS20 packet already containing denomination info

Reply:
yeah, won't quite a work, but would be good to reuse if we can do it cleanly

Reply:
it will be a bit, as the origin token denom on the cosmos chain is different than on Axelar. makes validation harder

Reply:
I think this is going to be a bit complicated to reuse. I would prefer to get this into the release as is and we can change this as part of the bigger refactoring we plan to do around GMP